src/ariadne_wallhug.c

```diff
@@ -2063,15 +2070,23 @@ static TbBool is_valid_hug_subtile(MapSubtlCoord stl_x, MapSubtlCoord stl_y, Pla
             return false;
         }
     }
+    if ((digflags & ToolDig_AllowLiquidWBridge) == ToolDig_AllowLiquidWBridge)
```

Comment:
I'm not sure if this is the correct place to add this functionality (I'm still reading some code to determine my opinion on the matter), but if it is, replace:
```
if ((digflags & ToolDig_AllowLiquidWBridge) == ToolDig_AllowLiquidWBridge)
{
    if (slab_good_for_computer_claim_path(slb))
    {
        return false;
    }
}
```
with 
```
if ((digflags & ToolDig_AllowLiquidWBridge) && slab_kind_is_liquid(slb->kind))
{
    return false;
}
```